### DISCUSSION TITLE
posts/%E4%BD%BF%E7%94%A8ollama%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2%E5%A4%A7%E6%A8%A1%E5%9E%8B

### DISCUSSION THREAD

**Discussion by giscus** [Peng-Hello/ginpang_domain#5] [Announcements]
# posts/%E4%BD%BF%E7%94%A8ollama%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2%E5%A4%A7%E6%A8%A1%E5%9E%8B

https://ginpang.netlify.app/posts/%E4%BD%BF%E7%94%A8ollama%E6%9C%AC%E5%9C%B0%E9%83%A8%E7%BD%B2%E5%A4%A7%E6%A8%A1%E5%9E%8B

**Comment by Peng-Hello** [comment]
这个是个很方便的工具，可以让我们快速本地部署各种开源大模型。组建本地 AI 工作流。点个赞 👍

**Comment by BAMDEVWEN** [comment]
谢谢博主的分享，有用
